OpenShift/helm/nexus-repository-manager/templates/NOTES.txt

```diff
@@ -0,0 +1,12 @@
+1. To access Nexus:
+
+   NOTE: It may take a few minutes for the ingress load balancer to become available or the backends to become HEALTHY.
+         You can watch the status of the backends by running:
+         `kubectl get ingress -o jsonpath='{.items[*].metadata.annotations.ingress\.kubernetes\.io/backends}'`
```

Comment:
I had noticed in my testing a while ago that the ingress does not yet automatically exist, but we have stories following very closely that will either make the ingress available automatically or add the docs to show how to do that. 👍